t_health_monitoring/test_health_monitor.py

```diff
@@ -190,22 +192,25 @@ def test(self):
         back2 = self.get_server("nginx2")
         back3 = self.get_server("nginx3")
         res = self.run_curl(REQ_COUNT)
-        self.assertTrue(list(set(res)) == [502], "No 502 in statuses")
+        self.assertEqual(res["502"], 100, "No 502 in statuses")
 
         # 2
         self.wait_for_server(back1)
         self.wait_for_server(back2)
         res = self.run_curl(REQ_COUNT)
-        self.assertTrue(sorted(list(set(res))) == [403, 404], "Not valid status")
+        self.assertEqual(res["403"], 50, "No 403 in statuses")
```

Comment:
I think the assert message is not clear. Is it kind-of `number of responses with returned code 403 is not equal to expected ones`?

This comment is related for all similar cases in the PR.